content/docs/uncontrolled-components.md

```diff
@@ -64,21 +65,21 @@ render() {
 }
 ```
 
-Likewise, `<input type="checkbox">` and `<input type="radio">` support `defaultChecked`, and `<select>` and `<textarea>` supports `defaultValue`.
+相同地， `<input type="checkbox">` 和 `<input type="radio">` 支援 `defaultChecked`，而 `<select>` 和 `<textarea>` 支援 `defaultValue`。
 
-## The file input Tag {#the-file-input-tag}
+## 檔案輸入標籤 {#the-file-input-tag}
 
-In HTML, an `<input type="file">` lets the user choose one or more files from their device storage to be uploaded to a server or manipulated by JavaScript via the [File API](https://developer.mozilla.org/en-US/docs/Web/API/File/Using_files_from_web_applications).
+在 HTML 裡，`<input type="file">` 讓使用者能夠選擇從他們的裝置儲存來上傳一個或多個檔案到伺服器，或由 JavaScript 透過 [File API](https://developer.mozilla.org/en-US/docs/Web/API/File/Using_files_from_web_applications) 來處理。
 
 ```html
 <input type="file" />
 ```
 
-In React, an `<input type="file" />` is always an uncontrolled component because its value can only be set by a user, and not programmatically.
+在 React 裡，`<input type="file" />` 永遠都是不可控制元件，因為它的值只能被使用者設定，而無法由程式碼來設定。
 
-You should use the File API to interact with the files. The following example shows how to create a [ref to the DOM node](/docs/refs-and-the-dom.html) to access file(s) in a submit handler:
+你應該使用 File API 來與檔案之間互動。以下範例顯示如何建立一個 [ref 到 DOM 節點上](/docs/refs-and-the-dom.html) 來取得在送出的 handler 的檔案：
 
 `embed:uncontrolled-components/input-type-file.js`
 
-[](codepen://uncontrolled-components/input-type-file)
+[在 CodePen 試試看](codepen://uncontrolled-components/input-type-file)
```

Comment:
```suggestion
[在 CodePen 試試看上](codepen://uncontrolled-components/input-type-file)
```

Don't add `**` symbol in the here, it will broken, ref #110